src/components/TitleTransferPanel/AddressInput/AddressInput.stories.mdx

```diff
@@ -0,0 +1,140 @@
+import { useState } from "react";
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks";
+import { AddressInput } from "./AddressInput";
+import { AddressButtons } from "./../AddressButtons";
+
+<Meta title="TitleTransferPanel/AddressInput" component={AddressInput} />
+
+# Address Input Single
+
+Address Input Single.
+
+---
+
+### Props
+
+<Props of={AddressInput} />
+
+---
+
+### Transfer
```

Comment:
Will suggest to name it such that it's not tied to the business case, in this case, maybe `Inlined`, `Block`, `Disabled`, and `Errored`